code-studio/src/js/code-studio.js

```diff
@@ -28,6 +28,10 @@ require('./components/send_to_phone.jsx');
 require('./components/share_dialog.jsx');
 require('./components/small_footer.jsx');
 require('./components/GridEditor.jsx');
+require('./components/Icon.jsx');
+require('./components/IconLibrary.jsx');
+require('./components/IconList.jsx');
+require('./components/IconListEntry.jsx');
```

Comment:
I think you may only need the top-level `IconLibrary` here, since it will require its dependencies.